docs/_docs/plugins.md

```diff
@@ -960,6 +960,7 @@ You can find a few useful plugins at the following locations:
 - [Jekyll-Post](https://github.com/robcrocombe/jekyll-post): A CLI tool to easily draft, edit, and publish Jekyll posts.
 - [jekyll-numbered-headings](https://github.com/muratayusuke/jekyll-numbered-headings): Adds ordered number to headings.
 - [jekyll-pre-commit](https://github.com/mpchadwick/jekyll-pre-commit): A framework for running checks against your posts using a git pre-commit hook before you publish them.
+- [jekyll-pwa-plugin](https://github.com/lavas-project/jekyll-pwa): A plugin provides PWA support for Jekyll. Generate a service worker and provides precache and runtime cache with Google Workbox.
```

Comment:
Can you change `Generate` to `Generates` please?

I'm still learning about PWAs but I wasn't aware that there was a dependency there on Google Workbox. Is that just an implementation detail of your plugin?

Reply:
@mattr- It's an implementation by Google to make it easier to get started.

Reply:
Thanks guys. As DirtyF says, Google Workbox has already developed a series of [tools](https://developers.google.com/web/tools/workbox/) . If you use Webpack or Gulp as your build tool, you can easily generate a service worker. But in my blog, I don't want to use even npm so I try to integrate this function in Jekyll build process.